k8s/namespaces/bsp/bulk-scan-processor/demo.yaml

```diff
@@ -3,8 +3,6 @@ kind: HelmRelease
 metadata:
   name: bulk-scan-processor
   annotations:
-    flux.weave.works/automated: "true"
-    flux.weave.works/tag.java: glob:pr-2080-*
     hmcts.github.com/prod-automated: disabled
```

Comment:
we can remove all `annotations`

Reply:
This will be removed during a follow up PR when demo environment 'prod-automated annotations: disabled' will be removed.  Need to keep for now till then.